tab_news.md

```diff
@@ -6,6 +6,10 @@ order: 3
 tags: mstg
 ---
 
+## October 4th, 2019: Pre-release of MASVS 1.2!
+
+We have released a first releaseclient of MASVS Version 1.2. This release-client will be the referenc document for further translations.
```

Comment:
Can we simplify? Not sure about the term release client, can we say:

We have a pre-release of MASVS Version 1.2. This will be the reference document for further translations.

Reply:
sure :D

Reply:
done